installutils/helminstall/installer.go

```diff
@@ -35,6 +35,9 @@ type Installer interface {
 }
 
 type InstallerConfig struct {
+	KubeConfig string
```

Comment:
super minor thing, but let's make sure that both fields get some brief documentation